ValidationRelay/ContentView.swift

```diff
@@ -46,8 +46,12 @@ struct ContentView: View {
                             // Connect or disconnect the relay
                             if newValue {
                                 relayConnectionManager.connect(getCurrentRelayURL())
+                                // Prevent the screen from turning off
+                                UIApplication.shared.isIdleTimerDisabled = true
```

Comment:
I don't think this is the right place for the timer to be disabled, because it is only triggered by the toggle: if the app is closed and reopened with the relay enabled already, it will automatically connect without the toggle, and never turn off the idle timer.